web/lib/classes/KYCClient.js

```diff
@@ -0,0 +1,213 @@
+import SerialPort from 'serialport'
+import Readline from '@serialport/parser-readline'
+import assert from "assert";
+
+const NUM_LEDS_PER_CHANNEL = 512;
+
+class MessageType {
+  constructor(name, byteValue) {
+    this.name = name;
+    this.byteValue = byteValue;
+  }
+
+  toString() {
+    return this.name
+  }
+}
+
+const MESSAGE_TYPES = {
+  // in
+  Ready: new MessageType("Ready", Buffer.from([0x08])),
+  Debug: new MessageType("Debug", Buffer.from([0x01])),
+  Pull: new MessageType("Pull", Buffer.from([0x02])),
+
+  // out
+  Data: new MessageType("Data", Buffer.from([0x01])),
+  Fire: new MessageType("Fire", Buffer.from([0x02])),
+  Swap: new MessageType("Swap", Buffer.from([0x10]))
+}
+
+
+const START_SEQ = Buffer.concat([Buffer.from([0xe5]), Buffer.from([0x6b]), Buffer.from([0x03]), Buffer.from([0x1d])]);
+
+class Sensor {
+  constructor(position) {
+    this.position = position
+    this.tension = 0
+    // it was always 10 in the old config
```

Comment:
Let's make it a par - we will need to (1) change them (2) have them different for different sticks right now